dp3t-sdk/sdk/build.gradle

```diff
@@ -23,7 +22,7 @@ android {
 		minSdkVersion 23
 		targetSdkVersion 30
 		versionCode 210
```

Comment:
Maybe it would make sense to extract the `versionCode` to the gradle.properties as well.

Reply:
True, since the version name is extracted, we could do the same here